test/module/irohad/model/command_validate_execute_test.cpp

```diff
@@ -84,15 +85,11 @@ class CommandValidateExecuteTest : public ::testing::Test {
         *command, *wsv_query, *wsv_command, creator.account_id);
   }
 
-  /// Throws exception if result does not contain value
-  void checkValueCase(const ExecutionResult &result) {
-    boost::get<ExecutionResult::ValueType>(result);
+  // return result with empty errpr message
```

Comment:
Use `///` for documentation strings.
Also, s/errpr/error.

Reply:
Okay, will fix